### PR TITLE
test: sync

### PR DESCRIPTION
**Pull request by joshuarli**

### CODE REVIEW

File: tests/test_sync.py [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+        "devenv.sync.run_procs", side_effect=[True, True, True]
+    ) as mock_run_procs:
+        reporoot = f"{tmp_path}/sentry"
+        venv = f"{tmp_path}/sentry/.venv"
+
+        os.makedirs(f"{reporoot}/devenv")
+        os.makedirs(venv)
+
+        with open(f"{venv}/pyvenv.cfg", "w") as f:
+            f.write("version = 3.8.16")
```
Comment (IanWoodard) [inline tests/test_sync.py:46]: Should we be hard coding this? It makes it harder to maintain

File: tests/test_sync.py [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
```
Comment (IanWoodard) [inline tests/test_sync.py:132]: Should we be hard coding this? It makes it harder to maintain

**Review by IanWoodard — APPROVED** [review]
One suggestion (in two places), other than that, I think we could abstract some of the code to reduce the amount of duplicate code but otherwise lgtm.

File: tests/test_sync.py (hunk 2 above)
Comment (joshuarli) [reply tests/test_sync.py:132]: this is fine and never needs to change because the main purpose here is to demonstrate venv recreation

**Pull request merged**